packages/web-twig/src/Twig/PropsExtension.php

```diff
@@ -112,12 +124,48 @@ public function renderStyleProp(Environment $environment, array $props): string
         ]);
     }
 
+    /**
+     * Converts a mixed value to a string.
+     *
+     * @param mixed $value The value to convert.
+     * @return string The value as a string.
+     */
+    private function toString($value): string
+    {
+        if (is_array($value)) {
+            // Convert array to JSON, use empty string on failure
+            return json_encode($value) ?: '';
+        }
+
+        if (is_bool($value)) {
+            // Convert boolean to 'true' or 'false'
+            return $value ? 'true' : 'false';
+        }
+
+        if ($value === null) {
+            // Handle null
+            return 'null';
+        }
+
+        // For other types that are safely convertible to string
+        if (is_scalar($value) || (is_object($value) && method_exists($value, '__toString'))) {
+            return strval($value);
```

Comment:
Can be fixed by `/** @var string $value */` with the correct type used.

Reply:
I removed the function at all 😬